specs/protocol/isthmus/configurability.md

```diff
@@ -39,10 +40,11 @@ The `ConfigType` enum represents configuration that can be modified.
 | `BASE_FEE_VAULT_CONFIG` | `uint8(1)` | Sets the Fee Vault Config for the `BaseFeeVault` |
 | `L1_FEE_VAULT_CONFIG` | `uint8(2)` | Sets the Fee Vault Config for the `L1FeeVault` |
 | `SEQUENCER_FEE_VAULT_CONFIG` | `uint8(3)` | Sets the Fee Vault Config for the `SequencerFeeVault` |
-| `L1_CROSS_DOMAIN_MESSENGER_ADDRESS` | `uint8(4)` | Sets the `L1CrossDomainMessenger` address |
-| `L1_ERC_721_BRIDGE_ADDRESS` | `uint8(5)` | Sets the `L1ERC721Bridge` address |
-| `L1_STANDARD_BRIDGE_ADDRESS` | `uint8(6)` | Sets the `L1StandardBridge` address |
-| `REMOTE_CHAIN_ID` | `uint8(7)` | Sets the chain id of the base chain |
+| `OPERATOR_FEE_VAULT_CONFIG` | `uint8(4)` | Sets the Fee Vault Config for the `OperatorFeeVault` |
```

Comment:
Given we are not combining this with L2 standard genesis, we are going to need to rethink how we go about configuring the system. How does the existing solidity implementation work?

Reply:
Seems like there was a bad merge in this file, I will have a look.

Currently I added a new method `setOperatorFeeScalars(uint32, uint64)` to `SystemConfig`. But reading the same file in the main branch, I see a section about [Fee Vault Config](https://github.com/ethereum-optimism/specs/blob/main/specs/protocol/isthmus/configurability.md#fee-vault-config), so I guess I should conform with that and add a `function setOperatoreFeeVaultConfig(address,uint256,WithdrawalNetwork)` method. Is it what you have in mind?

Reply:
Ah ok, I saw the Fee Vault Config section is part of L2 standard genesis, so I won't need to conform with that.